godtools/App/Flows/Menu/MenuFlow.swift

```diff
@@ -197,6 +194,39 @@ class MenuFlow: Flow {
             let copyrightInfoWebContent = CopyrightInfoWebContent(localizationServices: appDiContainer.localizationServices)
             
             navigateToWebContentView(webContent: copyrightInfoWebContent)
+            
+        case .deleteAccountTappedFromMenu:
+            
+            let viewModel = DeleteAccountViewModel(
+                flowDelegate: self,
+                localizationServices: appDiContainer.localizationServices
+            )
+            
+            let view = DeleteAccountView(viewModel: viewModel)
+            
+            let hostingView = DeleteAccountHostingView(view: view)
+                        
+            navigationController.pushViewController(hostingView, animated: true)
+            
+        case .backTappedFromDeleteAccount:
+            
+            navigationController.popViewController(animated: true)
+            
+        case .emailHelpDeskToDeleteOktaAccountTappedFromDeleteAccount:
+            
+            let finishedSendingMail = CallbackHandler { [weak self] in
+                self?.navigationController.dismiss(animated: true, completion: nil)
+            }
+            
+            let viewModel = MailViewModel(
+                toRecipients: ["help@cru.org"],
+                subject: "Please delete my account",
+                message: "I have created an account on the GodToolsapp and I would like to request that you delete my Okta account.",
```

Comment:
I'm assuming it's ok that this isn't translated.  Should there be a space in "GodToolsapp"?

Reply:
Added a space and for now we don't need to since account creation is English only.